mon-pix/app/routes/campaigns/fill-in-id-pix.js

```diff
@@ -9,7 +9,7 @@ export default BaseRoute.extend({
   session: service(),
 
   beforeModel(transition) {
-    this.set('campaignCode',transition.params['campaigns.fill-in-id-pix'].campaign_code);
+    this.set('campaignCode',transition.to.params.campaign_code);
```

Comment:
ça fonctionne ça ? transformer `transition.params['campaigns.fill-in-id-pix'].campaign_code` en `transition.to.params.campaign_code` me paraît risqué

Reply:
Plus que ça fonctionne, c'est ce qui est préconisé ! Cf. https://deprecations.emberjs.com/v3.x/#toc_transition-state

Reply:
C'est surtout : est-ce qu'on récupère le bon objet, vu qu'on ne précise plus le campaigns.fill-in-id-pix ?

Reply:
Oui, oui. Vérifié en live.